dask_cuda/tests/test_proxy.py

```diff
@@ -20,6 +21,12 @@
 from dask_cuda.proxify_device_objects import proxify_device_objects
 from dask_cuda.proxify_host_file import ProxifyHostFile
 
+if sys.version_info.minor < 8:
+    pytest.skip(
+        "Temporarily skipping some tests because of a bug "
+        "in Dask see <https://github.com/rapidsai/dask-cuda/issues/746>"
+    )
+
```

Comment:
```suggestion
pytest.mark.skipif(sys.version_info.minor < 8,
    reason="Temporarily skipping some tests because of a bug "
    "in Dask see <https://github.com/rapidsai/dask-cuda/issues/746>"
)
```